firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -555,9 +577,8 @@ void stopETBPins(void) {
 #endif /* EFI_PROD_CODE */
 
 void onConfigurationChangeElectronicThrottleCallback(engine_configuration_s *previousConfiguration) {
-	bool shouldResetPid = !etbController[0].etbPid.isSame(&previousConfiguration->etb);
-	for (int i = 0 ; i < ETB_COUNT; i++) {
-		etbController[i].shouldResetPid = shouldResetPid;
```

Comment:
Believe it or not, this used to be a race condition.  If the configuration was changed twice quickly back to back, and the ETB didn't need to update the second time, it would clear the flag even though it never reset the PID controller!